--- hpoj-0.90/apps/cmdline/Makefile.in.orig	2002-06-14 03:49:49.000000000 +0200
+++ hpoj-0.90/apps/cmdline/Makefile.in	2003-06-30 23:38:46.000000000 +0200
@@ -5,7 +5,7 @@
 DISTCLEANS=Makefile
 INCLUDES=Makefile ../../include/ptal.h ../../include/hpojip.h
 
-CFLAGS=-O -Wall -g @DEFINES_CMDLINE@ @INCLUDE_CMDLINE@ @LIBRARY_CMDLINE@ @VAR_RUN_PREFIX_CFLAGS@
+CFLAGS=$(OPTFLAGS) -Wall @DEFINES_CMDLINE@ @INCLUDE_CMDLINE@ @LIBRARY_CMDLINE@ @VAR_RUN_PREFIX_CFLAGS@
 
 all:		$(GOALS)
 
--- hpoj-0.90/apps/xojpanel/Makefile.in.orig	2002-06-13 02:40:44.000000000 +0200
+++ hpoj-0.90/apps/xojpanel/Makefile.in	2003-06-30 23:38:59.000000000 +0200
@@ -3,7 +3,7 @@
 MOC=@QT_MOC@
 
 LFLAGS = @LIBRARY_CMDLINE@ @LIBQT_CMDLINE@ -lptal
-CFLAGS = -O -Wall -g -I/usr/X11R6/include @INCLUDE_CMDLINE@
+CFLAGS = $(OPTFLAGS) -Wall -I/usr/X11R6/include @INCLUDE_CMDLINE@
 
 PIXMAPS = hpojlcd.xpm hpoj_lcdmon.xpm hpoj_mini.xpm
 COMMON_HEADERS = Makefile xojpanel.h
--- hpoj-0.90/lib/hpojip/Makefile.in.orig	2002-01-08 02:24:19.000000000 +0100
+++ hpoj-0.90/lib/hpojip/Makefile.in	2003-06-30 23:39:21.000000000 +0200
@@ -26,7 +26,7 @@
 # TODO: Detect system endianness in configure script!
 # For now, little-endian (Intel LSB-first) is assumed.
 
-CFLAGS=-O -Wall -g @DEFINES_CMDLINE@ @INCLUDE_CMDLINE@ @LIBRARY_CMDLINE@
+CFLAGS=$(OPTFLAGS) -Wall @DEFINES_CMDLINE@ @INCLUDE_CMDLINE@ @LIBRARY_CMDLINE@
 
 %.shared.o:	%.c $(INCLUDES)
 		$(CC) $(CFLAGS) -DHPOJIP_INTERNAL -fPIC -c -o $@ $<
--- hpoj-0.90/lib/ptal/Makefile.in.orig	2002-04-23 00:18:39.000000000 +0200
+++ hpoj-0.90/lib/ptal/Makefile.in	2003-06-30 23:39:39.000000000 +0200
@@ -17,7 +17,7 @@
 MLCDDIR=../../mlcd
 INCLUDES=Makefile $(INCLUDEDIR)/ptal.h ptal-internal.h $(MLCDDIR)/mlcd.h
 
-CFLAGS=-O -Wall -g @DEFINES_CMDLINE@ @INCLUDE_CMDLINE@ @LIBRARY_CMDLINE@ @VAR_RUN_PREFIX_CFLAGS@
+CFLAGS=$(OPTFLAGS) -Wall @DEFINES_CMDLINE@ @INCLUDE_CMDLINE@ @LIBRARY_CMDLINE@ @VAR_RUN_PREFIX_CFLAGS@
 
 %.shared.o:	%.c $(INCLUDES)
 		$(CC) $(CFLAGS) -fPIC -c -o $@ $<
--- hpoj-0.90/lib/sane/Makefile.in.orig	2002-07-10 03:13:39.000000000 +0200
+++ hpoj-0.90/lib/sane/Makefile.in	2003-06-30 23:39:52.000000000 +0200
@@ -16,7 +16,7 @@
 INCLUDEDIR=../../include
 INCLUDES=Makefile $(INCLUDEDIR)/ptal.h hpoj.h hpoj-tables.h sane.h saneopts.h
 
-CFLAGS=-O -Wall -g @DEFINES_CMDLINE@ @INCLUDE_CMDLINE@ @LIBRARY_CMDLINE@
+CFLAGS=$(OPTFLAGS) -Wall @DEFINES_CMDLINE@ @INCLUDE_CMDLINE@ @LIBRARY_CMDLINE@
 
 %.shared.o:	%.c $(INCLUDES)
 		$(CC) $(CFLAGS) -fPIC -c -o $@ $<
--- hpoj-0.90/mlcd/Makefile.in.orig	2002-04-23 00:18:39.000000000 +0200
+++ hpoj-0.90/mlcd/Makefile.in	2003-06-30 23:40:11.000000000 +0200
@@ -21,7 +21,7 @@
 LINKS=bp ex
 
 PWD=$(shell pwd)
-CFLAGS=-I$(PWD) -I$(PWD)/transport -O -g -Wall -DEX_TRANSPORT_UNIX_PORT -DPAR_PLATFORM_@PAR_PLATFORM@ -DUSB_PLATFORM_@USB_PLATFORM@ -DJD_DEBUGLITE @MLCD_CFLAGS@ @VAR_RUN_PREFIX_CFLAGS@
+CFLAGS=-I$(PWD) -I$(PWD)/transport $(OPTFLAGS) -Wall -DEX_TRANSPORT_UNIX_PORT -DPAR_PLATFORM_@PAR_PLATFORM@ -DUSB_PLATFORM_@USB_PLATFORM@ -DJD_DEBUGLITE @MLCD_CFLAGS@ @VAR_RUN_PREFIX_CFLAGS@
 
 %.o :		%.cpp $(INCLUDES)
 		$(CXX) $(CFLAGS) -c -o $@ $<
