? hpoj-0.8-ptal-hp.patch
Index: apps/cmdline/ptal-hp.c
===================================================================
RCS file: /cvsroot/hpoj/hpoj/apps/cmdline/ptal-hp.c,v
retrieving revision 1.19
retrieving revision 1.20
diff -c -r1.19 -r1.20
*** apps/cmdline/ptal-hp.c	2001/08/01 11:31:01	1.19
--- apps/cmdline/ptal-hp.c	2001/08/07 22:05:23	1.20
***************
*** 67,83 ****
  	/* Make sure the string is null-terminated in the right place. */
  	buffer[len]=0;
  
! #define PRINT_BRACKETS 0
! 
! 	printf(
! #if PRINT_BRACKETS
! 	"<"
! #endif
! 	"%s"
! #if PRINT_BRACKETS
! 	">"
! #endif
! 	"\n",buffer+start);
  }
  
  int handleDevice(ptalDevice_t dev,int arg,int argc,char **argv) {
--- 67,73 ----
  	/* Make sure the string is null-terminated in the right place. */
  	buffer[len]=0;
  
! 	printf("%s\n",buffer+start);
  }
  
  int handleDevice(ptalDevice_t dev,int arg,int argc,char **argv) {
